data/styles/clear/include/Icons.mapcss

```diff
@@ -1699,13 +1699,16 @@ area|z17-[power=station],
 node|z17-[power=plant],
 area|z17-[power=plant],
 /* Skip individual solar generators, because there are a lot of mapped panels */
-/* ToDo: Make separate wind generator icon? Because of a good landmark */
 node|z17-[power=generator][generator:source!=solar],
 area|z17-[power=generator][generator:source!=solar],
 node|z19-[power=substation],
 area|z19-[power=substation],
 {icon-image: power-m.svg;font-size: 11;}
 
+node|z17-[power=generator][generator:source=wind],
+area|z17-[power=generator][generator:source=wind],
+{icon-image: wind_turbine-m.svg;font-size: 11;}
```

Comment:
captions are not defined, so `font-size` is redundant
(or maybe consider adding captions on z18-)